app/build.gradle

```diff
@@ -22,6 +22,8 @@ dependencies {
     androidTestCompile 'com.android.support.test:rules:0.5'
     // Optional -- Hamcrest library
     androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
+
+    compile 'commons-io:commons-io:2.4'
```

Comment:
Is used by AudioUtils.saveWavToFile. It's more convenient with commons-io but totally doable without of course (just saves code)